docs/departments/fire-rescue/standard-operating-procedures.mdx

```diff
@@ -1,4 +1,4 @@
----
+a---
```

Comment:
Bug here?

Reply:
Please resolve this as soon as you can, so it can be merged into the PR.

Thanks!